src/languageservice/services/yamlCompletion.ts

```diff
@@ -225,19 +211,6 @@ export class YAMLCompletion extends JSONCompletion {
         }
 
         const location = Parser.getNodePath(node);
```

Comment:
Do you need to leave `location` here now that you have removed the `contributions.forEach`?